src/service/resources-impl.js

```diff
@@ -859,6 +871,19 @@ export class Resources {
     this.visibilityStateMachine_.setState(
       this.viewer_.getVisibilityState()
     );
+
+    if (this.maybeChangeHeight_) {
+      this.maybeChangeHeight_ = false;
+      this.vsync_.measure(() => {
```

Comment:
@jridgewell I would rather not do this optimization now, if in the future this become a performance problem we can recheck.